pypeit/find_objects.py

```diff
@@ -485,7 +490,10 @@ def global_skysub(self, skymask=None, update_crmask=True, trim_edg=(3,3),
                                           counts=previous_sky,
                                           count_scale=self.sciImg.img_scale,
                                           noise_floor=self.sciImg.noise_floor)
-            self.sciImg.ivar = utils.inverse(var)
+            skysub_ivar = utils.inverse(var)
+        else:
+            skysub_ivar = self.sciImg.ivar
+
```

Comment:
The reason why this was `self.sciImg.ivar = utils.inverse(var)` was because the `global_sky` computed here is also used in `skysub.local_skysub_extract`. So in `skysub.local_skysub_extract` the update `global_sky` is used, but the old `self.sciImg.ivar` is used, and this caused bad noise (if remember correctly higher) in the extracted objects. I wasn't able to pin point why that happened, but happened in the DEIMOS spectra that I was testing.

Reply:
Hey @jhennawi I tested the DEIMOS reduction on your branch and you are right the problem I was mentioning does not exists. Sorry for introducing this bug.

Reply:
Was not a bug. Just not a great idea to replace that ivar in place given that it is not correc. Thanks for doing this!